engine-nextgen/src/main/kotlin/graphql/nadel/NextgenEngine.kt

```diff
@@ -58,6 +64,7 @@ class NextgenEngine @JvmOverloads constructor(
     private val logNotSafe = getNotPrivacySafeLogger<NextgenEngine>()
     private val log = getLogger<NextgenEngine>()
 
+    private val coroutineScope = CoroutineScope(SupervisorJob() + Dispatchers.Default)
```

Comment:
Ok I'm sneaking in this change. Coroutines need a "scope" or an effectively an owner to be launched from. We were previously using `GlobalScope` which basically means nobody is managing the coroutines.

This isn't as necessary for us, since once a Nadel instance is "closed" by the AGG we don't send requests to it anymore.

But this can be useful for launching long standing tasks e.g. periodically doing something, but that something is tied to the Nadel instance.

This change is really just housekeeping.